agent/connect/ca/provider_vault_auth_azure.go

```diff
@@ -0,0 +1,153 @@
+package ca
+
+import (
+	"fmt"
+	"io/ioutil"
```

Comment:
`ioutil` has been deprecated so you'll probably want `io` here

Reply:
First I've seen of these that it wasn't os.ReadFile. Thanks.